### PR TITLE
mch on board repo

### PR DESCRIPTION
**Pull request by cameronmwall**
Signed-off-by: Cameron Wall <cwall@redhat.com>

### CODE REVIEW

File: ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  inputs:
+    builder-go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux-amd64
```
Comment (ray-harris) [inline ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml:17]: Change from 1.15 to 1.16

File: ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
+- as: e2e-cluster
+  steps:
+    dependencies:
+      COMPONENT_IMAGE_REF: backplane-operator
+    env:
+      CLUSTERPOOL_GROUP_NAME: acm-cicd-prow
+      CLUSTERPOOL_HOST_NAMESPACE: cicd
+      CLUSTERPOOL_HOST_PROW_KUBE_SECRET: acm-cicd-clusterpool-shared
```
Comment (ray-harris) [inline ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml:55]: You need to set up your own clusterpool secret for your squad and not use the cicd clusterpool secret.

File: ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+      from: bin
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: ocm-e2e-clusterpool
+- as: e2e-kind
+  steps:
+    env:
+      AWS_CREDENTIALS_SECRET: ocm-cicd-aws-kind
```
Comment (ray-harris) [inline ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml:70]: You need to set up your own aws secret for your squad and not the the cicd aws secret.

File: ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-main.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  to: backplane-operator
+promotion:
+  name: "1.0"
+  namespace: open-cluster-management
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```
Comment (ray-harris) [inline ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-main.yaml:31]: You need to add the following tests to your main branch:
- sonar-pre-submit
- sonar-post-submit
- e2e-cluster
- e2e-kind
- (optionally) premerge-image-mirror

Your actual testing will be done in the main branch while you're developing for 1.0, not in the backplane-1.0 branch.

When you cut to the next branch (say 2.0), main will do change to do testing for 2.0, but you'll still need the tests in the backplane-1.0 branch to test any z-stream releases.

File: ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-main.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  inputs:
+    open-cluster-management_builder_go1.15-linux-amd64:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux-amd64
```
Comment (ray-harris) [inline ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-main.yaml:21]: Change 1.15 to 1.16

**Review by ray-harris — COMMENTED** [review]
The rehearsal tests that are failing are fine, but you need to add an OWNERS file in the following locations:
- ci-operator/config/open-cluster-management/backplane-operator
- ci-operator/jobs/open-cluster-management/backplane-operator

You can just copy the one from here to those directories:
https://github.com/open-cluster-management/backplane-operator/blob/main/OWNERS

File: ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml (hunk 2 above)
Comment (cameronmwall) [reply ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml:55]: @ray-harris Am I able to just remove the e2e-cluster and e2e-kind tests? I noticed that my squad does not currently use them for any of the other repos, but that was included in David's example.

Comment (ray-harris) [reply ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml:55]: Yes, you can remove them. The examples are not just copy/paste. You need to know what applies to your repo and then how to adapt it.

**Review by ray-harris — COMMENTED** [review]
In both your main and backplane-1.0 configs, you have the pre-submit sonar scan, but you still need to add the post-submit sonar scan. It's the same check just with one more line added to make it post-submit and a different name. See:

https://github.com/open-cluster-management/cicd-docs/blob/main/prow/onboarding.md#go-sonarcloud-post-submit-scan

**Pull request merged**